### ISSUE TITLE
Microsoft sign-in with Chrome extension Manifest v3

### ISSUE THREAD

**Issue by rendomnet** [firebase/firebase-js-sdk#6567]
### [REQUIRED] Describe your environment

  * Operating System version: Mac Os
  * Browser version: Chrome latest
  * Firebase SDK version: 11.0.1
  * Firebase Product: auth

### [REQUIRED] Describe the problem
According to this documentation https://firebase.google.com/docs/auth/web/microsoft-oauth
firebase do not support manual sign-in flow
How to use Microsoft sign when developing Chrome extension with Manifest v3?

#### Steps to reproduce:
According to documentation there is no way for Chrome extension with Manifest v3 to sign in with Microsoft.

**Comment by sam-gc** [comment]
Thanks for filing this. It is true that the way social sign in worked under Manifest v2 no longer works in Manifest v3. We're aware of the issue but can't provide any timeline for updates/changes.

**Comment by rendomnet** [comment]
@sam-gc I cant event use my manual oauth0 auth flow it says "invalid credentials".

**Comment by sam-gc** [comment]
Yeah unfortunately that's a limitation of Microsoft sign in. This works for other providers but not for Microsoft. See the section in the docs entitled "Advanced: Handle the sign-in flow manually" -- https://firebase.google.com/docs/auth/web/microsoft-oauth#advanced:-handle-the-sign-in-flow-manually"
